src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx

```diff
@@ -124,10 +125,11 @@ export const RemoveLiquidityInput: React.FC<RemoveLiquidityInputProps> = ({
   const [shareOfPoolDecimalPercent, setShareOfPoolDecimalPercent] = useState<string | undefined>()
   const [shouldShowWarningAcknowledgement, setShouldShowWarningAcknowledgement] = useState(false)
 
-  const { assetId, type: opportunityType } = useMemo(
+  const { assetId, type: _opportunityType } = useMemo(
     () => fromOpportunityId(opportunityId),
     [opportunityId],
   )
+  const [opportunityType, setOpportunityType] = useState<AsymSide | 'sym'>(_opportunityType)
```

Comment:
```suggestion
  const [opportunityType, setOpportunityType] = useState<OpportunityType>(_opportunityType)
```

Reply:
Changed in [3f46c39](https://github.com/shapeshift/web/pull/7351/commits/3f46c39c62ae8b335139b436a73c7c22405becd5)